src/v/kafka/server/group.cc

```diff
@@ -52,6 +52,7 @@ group::group(
   config::configuration& conf,
   ss::lw_shared_ptr<cluster::partition> partition,
   ss::sharded<cluster::tx_gateway_frontend>& tx_frontend,
+  ss::sharded<features::feature_table>& feature_table,
```

Comment:
Would it be possible to add a little bit of context to the commit message ?

Reply:
+1